docs/DeveloperGuide.adoc

```diff
@@ -117,9 +118,15 @@ image::EnqueueSequenceDiagram.png[]
 .Interactions inside the Logic Component for the `next 1` Command
 image::NextSequenceDiagram.png[]
 
+Given below is the Sequence Diagram for interactions within the `Logic` component for the `execute("cancelappt 1")` API call.
+
+.Interactions Inside the Logic Component for the `cancelappt 1` Command
+image::CancelAppointmentSequenceDiagram.png[]
+
+NOTE: The lifeline for `CancelAppCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
```

Comment:
Perhaps you can consider using Photoshop to remove the excess dotted line instead of putting a comment about it.